spec/services/importers/new_cases_spec.rb

```diff
@@ -2,8 +2,21 @@
 
 RSpec.describe Importers::NewCourtCase do
   describe '#perform' do
-    it 'add specs' do
-      skip
+    it 'perform correctly finds or creates and returns a new case' do
+      create(:county, name: 'Oklahoma')
+      create(:case_type, :felony)
+      create(:county, name: 'Arkansas')
+      create(:case_type, :misdemeanor)
+      file_path = 'spec/fixtures/importers/link.html'
+      html = File.read(file_path)
+      link_html = Nokogiri::HTML.parse(html)
+      link_html = link_html.css('a').first
+
+      data = described_class.new(link_html)
+      data.perform
+      expect(data.case_number).to eq 'CF-2021-489'
+      expect(data.case_types.keys.first).to eq 'CF'
+      expect(data.counties.keys.first).to eq 'Oklahoma'
     end
```

Comment:
I'm thinking there could be two different specs here:

1) Test that it creates the case when it does not exist in the database (What you currently have)
2) Test that it does not duplicate a case if it already exists in the database (Different spec)